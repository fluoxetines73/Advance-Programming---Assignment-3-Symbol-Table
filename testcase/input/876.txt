BEGIN
END
RPRINT
INSERT z number
INSERT b string
LOOKUP z
ASSIGN b 'Test123'
INSERT d number
RPRINT
PRINT
ASSIGN z 456 random
BEGIN
BEGIN
RPRINT
BEGIN
PRINT
RPRINT
END
PRINT
END
END