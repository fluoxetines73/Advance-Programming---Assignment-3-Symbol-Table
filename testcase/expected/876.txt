Invalid: ASSIGN z 456 random